### ISSUE TITLE
Seid update auf 10.5.6 funktioniert das automatische be- und entladen nicht mehr

### ISSUE THREAD

**Issue by Pipel19** [Stephan-S/FS19_AutoDrive#478]
Hallo zusammen,

seit dem update auf 10.5.6 funktioniert das automatische be- und entladen nicht mehr. Der Fahrer fährt einfach durch. Getestet mit Gärresten. Er öffnet kurz den Deckel des Anhängers, genutzt wird ein lsfm_Universal_TankPack Hänger. Bei älteren Versionen ging es damit noch ohne Probleme. 
Des weiteren hält Autodrive bei nach Beladen von Silage aus dem FS19_FEDmods_Fermenter an und fährt nicht weiter. Dabei springt die Anzeige die ganze Zeit hin und her im AD hud. Ging bei älteren Versionen auch ohne Probleme.
Ich habe mal die Log Datei beigefügt, da ich nicht mehr weiter weiß. Wenn jemand eine Lösung hat wäre ich sehr dankbar. Wird noch mehr benötigt, um etwas sagen zu können?
[log.txt](https://github.com/Stephan-S/FS19_AutoDrive/files/3592336/log.txt)

**Comment by Roby1164** [comment]
Let's start with that you have a big mistake at the very beginning:
Error: Failed to open xml file 'C: / Users / phili / Documents / My Games / FarmingSimulator2019 / mods / AutoDrive_NF_Marsch_oG / modDesc.xml'.

**Comment by Huskiel** [comment]
Habe auch das Problem das der Fahrer beim beladen nicht hält.
Habe es mit Gras und Silage probiert.

**Comment by Pipel19** [comment]
Hi Roby, most of the errors in log I eliminate now. But the problem still there. Do you have another idea where it comes from? I attached the actual log. What else could be the reason?
[log.txt](https://github.com/Stephan-S/FS19_AutoDrive/files/3596343/log.txt)

**Comment by Pipel19** [comment]
Yes, I already did this. Tanker trailer opens the hatch, so it seems that AI sees the trigger. But doesn't stop.
Try it now also with 10.5.1. With this Version it seems to work. So it seems that it happen between 10.5.1 and 10.5.5. Any idea what I'm doing wrong? 
Here is the log with Version 10.5.1
[log.txt](https://github.com/Stephan-S/FS19_AutoDrive/files/3596773/log.txt)

**Comment by Roby1164** [comment]
> Hi Roby, most of the errors in log I eliminate now. But the problem still there. Do you have another idea where it comes from? I attached the actual log. What else could be the reason?
> [log.txt](https://github.com/Stephan-S/FS19_AutoDrive/files/3596343/log.txt)

First thing, remove 1.0.5.6, which is not good, and put 1.0.5.5, or go back to 1.0.5.1 for the moment, which is the one that surely doesn't have these problems

**Comment by Stephan-S** [comment]
Hey guys, I think this issue is already fixed. Not in the current release 1.0.5.6, but If you zip the code yourself, it will contain the fix.
I will update it in the weekend to a new release.

**Comment by Roby1164** [comment]
> Hey guys, I think this issue is already fixed. Not in the current release 1.0.5.6, but If you zip the code yourself, it will contain the fix.
> I will update it in the weekend to a new release.

I don't know, in 1.0.5.6, there is also the problem of covers that no longer open from the second load onwards.
For the moment, the one that is going is the 1.0.5.5

**Comment by Pipel19** [comment]
Thanks for your help. I will try this.
And what I fortgot to mention. Great mod, thanks a lot. makes really fun to play the game with this mod
